dev-cpp/cpp-taskflow/cpp-taskflow-3.0.0.ebuild

```diff
@@ -12,42 +12,30 @@ SRC_URI="https://github.com/${PN}/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
 LICENSE="MIT"
 SLOT="0"
 KEYWORDS="~amd64 ~x86"
-IUSE="doc"
-
-RDEPEND=""
-DEPEND=""
+IUSE="examples doc test"
+RESTRICT="!test? ( test )"
 
 S="${WORKDIR}/taskflow-${PV}"
-
-PATCHES=(
-)
-
 HTML_DOCS=( docs/. )
 
 src_prepare() {
-	default
-
 	# fix library directoy
 	sed -i "s#/lib#/$(get_libdir)#g" CMakeLists.txt || die "sed failed"
 
-	cmake-utils_src_prepare
+	cmake_src_prepare
 }
 
 src_configure() {
-	# FIXME: enable CUDA and TESTS via use flag
+	# TODO enable CUDA via USE flag
 	local mycmakeargs=(
 		-DTF_BUILD_CUDA=OFF
-		-DTF_BUILD_TESTS=OFF
-		-DTF_BUILD_EXAMPLES=OFF
+		-DTF_BUILD_EXAMPLES=$(usex examples)
 	)
 
-	cmake-utils_src_configure
+	cmake_src_configure
 }
 
 src_install() {
-	cmake-utils_src_install
-
-	if $(use doc); then
-		einstalldocs
-	fi
+	cmake_src_install
+	use doc && einstalldocs
```

Comment:
this is dead code, as `cmake_src_install` calls `einstalldocs` internally

Reply:
Ah that's right. Removed